packages/ember-views/lib/mixins/view_support.js

```diff
@@ -538,7 +538,16 @@ export default Mixin.create({
       let owner = getOwner(this);
       let dispatcher = owner && owner.lookup('event_dispatcher:main');
 
-      if (dispatcher && dispatcher.canDispatchToEventManager === null) {
+      deprecate(
+        `[DEPRECATED] \`canDispatchToEventManager\` has been deprecated.`,
```

Comment:
Let's have this message mention `eventManager` instead of `canDispatchToEventManager`, and also add the ${this} to the interpolation (so it's clear to someone seeing the deprecation which component caused it.